site-manage/site-group-manager/src/main/webapp/WEB-INF/templates/fragments/javascript.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<body>
+<script th:fragment="indexJs" th:inline="javascript">
+  /*<![CDATA[*/
+  $('#delete-groups-cancel-button').click(function(e) {
+    e.preventDefault;
+    window.location = window.location.href.replace('/sakai-site-group-manager.helper', '');
+  });
+
+  function checkSubmitButton() {
+    var anyCheckboxEnabled = false;
+      $(':checkbox').each(function() {
```

Comment:
Indentation a bit odd.

Reply:
Yeah, small mistake, after the indentation of 3K lines of code I made a mistake.

I'll do a small PR fixing this